actionlist/app/containers/PreferencesContainer.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native';
+import Button from 'react-native-button';
+import { Actions } from 'react-native-router-flux';
+import Header from '../components/header';
+import Dropdown from '../components/Dropdown';
+import { Colors } from '../lib/commons';
+
+const styles = StyleSheet.create({
+  componentContainer: {
+    backgroundColor: Colors.Grey,
+    marginTop: 10,
+    marginRight: 10,
+    marginLeft: 10,
+    padding: 10,
+    alignItems: 'center',
+    borderRadius: 8,
+    borderStyle: 'solid',
+    borderWidth: 1,
+
+  },
+  text: {
+    color: Colors.IUCrimson,
+    marginLeft: 10,
+  },
+  preferencesContainer: {
+    marginTop: 10,
+    backgroundColor: 'grey',
+    marginRight: 10,
+    marginLeft: 10,
+    borderRadius: 8,
+    borderStyle: 'solid',
+    borderWidth: 1,
+  },
+  preferenceSelect: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    marginBottom: 13,
+    marginLeft: 20,
+    marginRight: 10,
+    marginTop: 12,
+  },
+  preferenceText: {
+    fontSize: 15,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  preferences_buttons: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    marginTop: 10,
+    marginLeft: 10,
+    marginRight: 10,
+    justifyContent: 'space-between',
+  },
+
+  buttonContainer: {
+    borderRadius: 8,
+    width: 110,
+  },
+  preferencesButton: {
+    borderRadius: 8,
+    backgroundColor: Colors.IUCrimsonDark,
+    marginRight: 10,
+    marginLeft: 10,
+    padding: 5,
+    fontSize: 15,
+    color: '#ffffff',
+    height: 30,
+    alignItems: 'center',
+  },
+});
+
+const PreferencesContainer = () => (
+  <View>
+    <Header />
+    <View style={styles.componentContainer}>
+      <Text style={styles.preferenceText}>Preferences</Text>
+    </View>
+    <View style={styles.preferencesContainer}>
+      <View style={styles.preferenceSelect}>
```

Comment:
React components can take arrays as children, so instead of writing all of these out, why not have an array of document statuses (e.g., `const documentStatuses = ['saved', 'initiated', ...];`) and then make the children of the `View` here be something like `{ documentStatuses.map(documentStatus => <View ...><Text ...>{documentStatus}</Text></View> }`